src/classes/BDI_MigrationMappingUtility.cls

```diff
@@ -198,19 +202,22 @@ public class BDI_MigrationMappingUtility {
      * @description Updates Data Import custom settings
      * @param fieldMappingMethod String that determines what data import field mapping method
      * to use (i.e. help text, custom metadata)
-     * @param migrationType String that determines what field mapping set we're using (i.e. default set, custom migrated set)
      * @param status Deployment status
      */
-    public static Data_Import_Settings__c upsertCustomSettings(String fieldMappingMethod, String migrationType, String status) {
+    public static Data_Import_Settings__c upsertCustomSettings(String fieldMappingMethod, String status) {
         Data_Import_Settings__c dataImportSettings = UTIL_CustomSettingsFacade.getDataImportSettings();
         dataImportSettings.Field_Mapping_Method__c = fieldMappingMethod;
 
-        if (migrationType == CUSTOM_MAPPING) {
+        List<Data_Import_Field_Mapping_Set__mdt> fieldMappingSets = [SELECT Id
+            FROM Data_Import_Field_Mapping_Set__mdt
+            WHERE DeveloperName = :MIGRATED_CUSTOM_FIELD_MAPPING_SET_RECORD_NAME];
+
+        Boolean customFieldMappingSetExists = fieldMappingSets.size() > 0 ? true : false;
```

Comment:
This local variable has the same name as the class variable - if you truly need both is it possible to make the distinction clear in the names?